content/tools/sdk/MavenPlugin.md

```diff
@@ -61,12 +60,26 @@ An example configuration for the Contrast Maven Plugin:
     <configuration>
         <username>test_user</username>
         <apiKey>testApiKey</apiKey>
-        <serviceKey>testServiceKEy</serviceKey>
+        <serviceKey>testServiceKey</serviceKey>
         <apiUrl>http://app.contrastsecurity.com/Contrast/api</apiUrl>
         <orgUuid>QWER-ASDF-ZXCV-ERTY</orgUuid>
-        <appName>VehicleMPG-Master</appName>
-        <serverName>jenkins.slave1</serverName>
+        <appName>MyAppName</appName>
+        <serverName>MyServerName</serverName>
         <minSeverity>High</minSeverity>
     </configuration>
 </plugin>
 ```
+
+## Use the agent
+
+The final step is to add the agent to the JVM arguments used when the application is run. You must specify the following properties:
+
+```
+-javaagent:/target/contrast.jar -Dcontrast.appname=MyAppName -Dcontrast.server=MyServerName
```

Comment:
@scorlett , do we have customers always setting appname now?

Reply:
This is just when using the Maven plugin, which checks the vulns in TeamServer and will pass/fail the build based on what's been found. You have to specify the appName of the application that you want the plugin to check on the TeamServer side. So they don't _have_ to specify the appname when you run the agent, it's just so much easier if they do as they can set the Maven plugin config to the same thing.